src/seedu/addressbook/storage/jaxb/AdaptedPerson.java

```diff
@@ -9,12 +9,7 @@
 
 import seedu.addressbook.common.Utils;
 import seedu.addressbook.data.exception.IllegalValueException;
-import seedu.addressbook.data.person.Address;
-import seedu.addressbook.data.person.Email;
-import seedu.addressbook.data.person.Name;
-import seedu.addressbook.data.person.Person;
-import seedu.addressbook.data.person.Phone;
-import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.*;
```

Comment:
Although it may seem like a convenient shortcut to just import `.*`, this violates our coding standards (which you should adhere to) and is bad practice especially for big projects where a package can contain many other classes which you do not need to use here